db/migrate/20190607014201_add_column_approved_to_institutions.rb

```diff
@@ -0,0 +1,6 @@
+class AddColumnApprovedToInstitutions < ActiveRecord::Migration
+  def change
+    # Note this is going to result in null values, three-state-boolean problem
+    add_column :institutions, :approved, :boolean
```

Comment:
Following the comments above, you can set the existing records to `true` when you run the migration:
```suggestion
    add_column :institutions, :approved, :boolean, null: false, default: false
    reversible do |dir|
      dir.up do
        Institution.update_all(approved: true)
      end
    end
```

Reply:
Will do, thanks